pages/programs/youth-managed-farms.css

```diff
@@ -0,0 +1,168 @@
+/* Importing stuff! */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+
+/*Now the actual code */
+
+/* Grid Items */
+.grid-st {
+
+}
+.grid-mid {
+
+}
+.grid-end {
+	grid-column-start:2;
+	grid-column-end: 5;
+}
+/* Header Items */
+
+#headerBR {
+	display: grid;
+  	grid-template-columns: auto auto auto auto;
+	padding: 10px;
+	width:  75%;
+	margin: auto 15% auto 20%;
+	background-color: white;
+
+}
+
+#navBR {
+	margin-left: 20%;
```

Comment:
This margin is too big. This could be a case where we should put a `max-width` on the parent container to reduce the chance of this rule being needed.